TipCatDotNet.Api/Services/HospitalityFacilities/StripeAccountService.cs

```diff
@@ -0,0 +1,150 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CSharpFunctionalExtensions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using Stripe;
+using TipCatDotNet.Api.Data;
+using TipCatDotNet.Api.Models.HospitalityFacilities;
+using TipCatDotNet.Api.Options;
+
+namespace TipCatDotNet.Api.Services.HospitalityFacilities
+{
+    public class StripeAccountService : IStripeAccountService
+    {
+        public StripeAccountService(AetherDbContext context, Stripe.AccountService accountService, IOptions<StripeOptions> stripeOptions)
+        {
+            _context = context;
+            _accountService = accountService;
+            _stripeOptions = stripeOptions;
+        }
+
+
+        public async Task<Result<string>> Add(int memberId, MemberRequest request, CancellationToken cancellationToken)
```

Comment:
Can't we get the member's ID from the request?

Reply:
Yes, in this case i need to create constructor for this. I'll do it